lab-geoff/lib/requestHandler.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+
+module.exports = function(req, callback) {
+  req.on('data', function(data) {
```

Comment:
You're going to want to build this out a little more. The framework is on the right path though. `on` data, you want to add (`=+`) data to a body variable `let body = ''`. 

You also want to take into account a `on` end event, and when that event if triggered, you can take the completed body string and pass it out to your callback.